lib/rubocop/cop/rspec/capybara/visibility_matcher.rb

```diff
@@ -30,18 +30,11 @@ class VisibilityMatcher < Base
           MSG_FALSE = 'Use `:all` or `:hidden` instead of `false`.'
           MSG_TRUE = 'Use `:visible` instead of `true`.'
           CAPYBARA_MATCHER_METHODS = %i[
-            have_selector
-            have_css
-            have_xpath
-            have_link
-            have_button
-            have_field
-            have_select
-            have_table
-            have_checked_field
-            have_unchecked_field
-            have_text
-            have_content
+            have_selector have_css have_xpath have_link have_button
```

Comment:
might be a good idea to have an alphabetical order of the values

Reply:
In a simple reordering, `have_no_**` comes in the middle, so I reordered `have_**` and `have_no_**` respectively